modules/apps/translation/translation-service/src/main/java/com/liferay/translation/internal/info/item/updater/JournalArticleInfoItemFieldValuesUpdaterImpl.java

```diff
@@ -192,6 +217,13 @@ private String _getTranslatedContent(
 						targetLocale, targetLocale, field.getDataType(),
 						entry.getValue()));
 			}
+			else {
```

Comment:
@AliciaGarciaGarcia this can be simplified with an `else if`. I don't know if Brian will merge it like this. It's up to you to try. Feel free to forward.